signup-service/src/main/java/io/mosip/signup/services/CacheUtilService.java

```diff
@@ -390,4 +394,8 @@ private boolean isScriptNotLoaded(String scriptHash) {
         return scriptExists == null || !scriptExists.get(0);
     }
 
+    @Cacheable(value = UI_SPEC, key = "'ui_spec'")
+    public JsonNode getSchema() {
```

Comment:
This is pulling UI spec not the identity schema. I think we should rename this method @Md-Humair-KK